infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/KubernetesServerResolverTest.java

```diff
@@ -65,7 +66,8 @@
             Pair.of("http-server", new ServerConfigImpl("3054", "http", "/api", ATTRIBUTES_MAP)));
 
     KubernetesServerResolver serverResolver =
-        new KubernetesServerResolver(singletonList(nonMatchedByPodService), singletonList(ingress));
+        new KubernetesServerResolver(
+            new NoopPathDemangler(), singletonList(nonMatchedByPodService), singletonList(ingress));
```

Comment:
There seem to be no tests for the `PathDemangler`.

Reply:
Whoops, added.